packages/nx/src/utils/package-json.ts

```diff
@@ -142,15 +144,27 @@ let packageManagerCommand: PackageManagerCommands | undefined;
 export function getMetadataFromPackageJson(
   packageJson: PackageJson
 ): ProjectMetadata {
-  const { scripts, nx } = packageJson ?? {};
+  const { scripts, nx, description } = packageJson ?? {};
```

Comment:
We should think about if we want to allow spec'ing description in project.json directly as well. If so, this wouldn't be on metadata but rather just the project configuration itself.

Right now there's not a way to spec a description for non-js projects, so I think we should consider it.